### PR TITLE
SAIC-425 Detailed description for CF configuration file.

### PR DESCRIPTION
**Pull request by archyufa**
Updates made to config.ini based on improvements from SAIC-425: Detailed description for CF configuration file.

### CODE REVIEW

**Comment by roman-verchikov** [comment]
please rebase

File: configs/config.ini [hunk 1]
```diff
@@ -68,67 +155,186 @@ temp = /root/temp
...
 
+
+
+
 [src_objstorage]
+
+# Service name for object storage.
 service = swift
 
+
```
Comment (roman-verchikov) [inline configs/config.ini]: please remove spaces here

File: configs/config.ini [hunk 2]
```diff
@@ -68,67 +155,186 @@ temp = /root/temp
...
+
+
 [src_network]
+
+# Name service for network. Value: "auto" which detects available service.
 service = auto
 
+
```
Comment (roman-verchikov) [inline configs/config.ini]: please remove spaces here

File: configs/config.ini [hunk 3]
```diff
@@ -68,67 +155,186 @@ temp = /root/temp
...
+
+
 [src_identity]
+
+# Service name for Identity service.
 service = keystone
 
+
```
Comment (roman-verchikov) [inline configs/config.ini]: please remove spaces here

File: configs/config.ini [hunk 4]
```diff
@@ -68,67 +155,186 @@ temp = /root/temp
...
+# Database user password for the Glance service.
+# To get value run: #cat /etc/glance/glance-api.conf | grep mysql
 password = <glance_db_password>
+
+# Glance service database name. 
+# To get value run: #cat /etc/glance/glance-api.conf | grep mysql
 database_name = glance
 
+
```
Comment (roman-verchikov) [inline configs/config.ini]: please remove spaces here

**Comment by miarmak** [comment]
Please squash 2 commits into 1 and rebase.

**Comment by roman-verchikov** [comment]
@archyufa this pull request need rebase, please do

**Comment by roman-verchikov** [comment]
Followed in #404

**Pull request closed without merging**